### PR TITLE
588 start case views

### PR DESCRIPTION
**Pull request by rachaelparris**
screen shot 2019-02-21 at 9 08 43 am
screen shot 2019-02-21 at 9 08 47 am

### CODE REVIEW

File: web-client/src/views/StartCase/CaseTypeSelect.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import { connect } from '@cerebral/react';
+import { sequences, state, props } from 'cerebral';
+import React from 'react';
+
+export const CaseTypeSelect = connect(
```
Comment (codyseibert) [inline web-client/src/views/StartCase/CaseTypeSelect.jsx:5]: 👍

File: web-client/src/views/StartCase.jsx [hunk 2]
```diff
@@ -513,128 +512,147 @@ export default connect(
...
-                        aria-label="day, two digits"
-                        id="date-of-notice-day"
-                        name="day"
-                        type="number"
-                        min="1"
-                        max="31"
+                        id={`hasNotice-${hasNotice}`}
+                        type="radio"
+                        name="hasNotice"
+                        value={hasNotice}
```
Comment (codyseibert) [inline web-client/src/views/StartCase.jsx]: is the state going to be form.hasNotice: 'Yes', or will it be form.hasNotice: true?  I'd say verify we store true / false and not 'Yes' / 'No'

**Review by kkoskelin — APPROVED** [review]

File: web-client/src/views/StartCase.jsx [hunk 3]
```diff
@@ -513,128 +512,147 @@ export default connect(
...
-                        aria-describedby="date-of-notice-legend"
-                        aria-label="day, two digits"
-                        id="date-of-notice-day"
-                        name="day"
-                        type="number"
-                        min="1"
-                        max="31"
+                        id={`hasNotice-${hasNotice}`}
+                        type="radio"
+                        name="hasNotice"
```
Comment (codyseibert) [inline web-client/src/views/StartCase.jsx]: could you rename `hasNotice` to `hasIrsNotice` please?  In the Case.js model we already have a field called `irsNoticeDate`, so `hasIrsNotice` might be more consistent.  We should probably go into the Case.js file and add that to validation: 

`hasIrsNotice: joi.boolean().optional(),`

**Review by codyseibert — APPROVED** [review]

**Pull request merged**